src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -157,6 +157,10 @@ class SpanMRI(Enum):
     HTTP_ERROR_RATE_LIGHT = "e:spans/http_error_rate_light@ratio"
 
 
+class ErrorsMRI(Enum):
+    EVENT_INGESTED = "c:errors/event_ingested@none"
```

Comment:
I believe `errors` should be `escalating_issues` instead here. The second component in that MRI should be the `use_case_id`, which is `escalating_issues`